asdf_compression/blsc/compressor.py

```diff
@@ -7,10 +7,34 @@ class BloscCompressor(Compressor):
     def compress(self, data, **kwargs):
         import blosc
 
-        yield blosc.compress(data, **kwargs)
+        # Type size, necessary for shuffle filters
+        typesize = data.itemsize
+
+        # Compression level
+        clevel = kwargs.get("level", 9)
```

Comment:
I'd prefer to keep this as `clevel` to match the [blosc.compress signature](https://www.blosc.org/python-blosc/reference.html#blosc.compress). As the default already 9 I'd say let's let `**kwargs` handle the `clevel`.

Reply:
I was assuming that there is a (small) standard set of arguments that `asdf` supports, and that people could use these without knowing details about the compressor.

Apparently that's not the case, it's `level` for `zlib`, `compresslevel` for `bzip2`, etc...

Reply:
It's a bit of the "wild west" when it comes to compression keyword arguments. I'd be open to suggestions if you have any ideas for a less "leaky" api. In general compression has not received much attention in asdf (until now).